packages/babel-helper-define-polyfill-provider/package.json

```
@@ -19,6 +19,13 @@
     "./lib/node/dependencies.js": "./lib/browser/dependencies.js",
     "./src/node/dependencies.js": "./src/browser/dependencies.js"
   },
+  "exports": [
+    {
+      "import": "./esm/index.mjs",
+      "default": "./lib/index.js"
+    },
+    "./lib/index.js"
+  ],
```

@ljharb Is this correct to ensure compatibility?

Since this package is in 0.x it's ok to disallow imports to other internal files, but I want it to work on every Node.js version since it's what the main Babel packages do.

---

Yes, exactly! altho i usually do:
```suggestion
  "exports": {
    ".": [
      {
        "import": "./esm/index.mjs",
        "default": "./lib/index.js"
      },
      "./lib/index.js"
    ]
  },
```
I’m not sure if it makes a difference (it shouldn’t, but i haven’t tested the top-level being an array).

however, since you almost certainly want to include “./package.json” here, for tooling, you’d need the dot form anyways, so you could add package.json